07LessonSeven/in-class-project/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+
+		<title>NASA</title>
+		<link rel="stylesheet" href="css/my-style.css">
+
+	</head>
+
+	<body>
+		<header>
+
+			<!--White font color in middle of image  -->
+
+			<p class="text_Nasa">NASA</p>
+
+		</header>
+		<div class="divido">
+		</div>
+		<footer class="cleardiv">
+
+			<div class="block_1">
+				<!--First Float  -->
+
+				<h1>Social Media</h1>
+				<br> Twitter -@nasa
+				<br> Facebook fb.com/nasa
+				<br> instagram - @nasa
+				<br> Youtube - youtube.com/NASA
+				<br>
+
+			</div>
+			<!--Second Float  -->
+
+
+			<div class="block_2">
+				<h1>Office Info</h1>
+				<br> Public Communications Office
+				<br> NASA Headquarters
+				<br> 300 E. Street SW Suite Sr 30
+				<br> Washington, DC
+				<br> 20546
+				<br> (202)358-0001 (Office)
+				<br> (202)358-4338 (Fax)
+				<br>
+
+			</div>
+
+			<div class="block_3">
+				<!--Third Float and Contact Form -->
+
+
+				<h1>Contact Us</h1>
+				<br> Name:
+				<br>
+				<br>
+				<textarea>
+
+				</textarea>
+				<br>
+				<br>
```

Comment:
br's do work, but consider adding margin or padding as this will look a bit more professional.